osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -4,256 +4,262 @@
+    #     'default': 'pseudomerkator',
+    #     'groups': [
+    #         {
+    #             'name': 'Global coordinate reference systems',
+    #             'values': [
+    #                 {'name': 'pseudomerkator', 'label': 'Pseudo merkator'},
+    #                 {'name': 'wgs72', 'label': 'WGS 72'},
+    #                 {'name': 'wgs84', 'label': 'WGS 84'}
+    #             ]
+    #         },
+    #         {
+    #             'name': 'UTM zones for your export',
+    #             'values': [
+    #                 {'name': 'utm32', 'label': 'UTM zone 32'},
+    #                 {'name': 'utm33', 'label': 'UTM zone 33'}
+    #             ]
+    #         }
+    #     ]
+    # },
+    # 'detail_level': {
+    #     'label': 'Detail level',
+    #     'type': 'choice',
+    #     'default': 'verbatim',
+    #     'values': [
+    #         {'name': 'verbatim', 'label': 'Verbatim'},
+    #         {'name': 'simplified', 'label': 'Simplified'}
+    #     ]
+    # }
+}
 
-    @staticmethod
-    def export_formats():
-        return {
-            'spatialite': {
-                'name': 'SpatiaLite (SQLite)',
-                'file_extension': 'sqlite'
-            },
-            'gpkg': {
-                'name': 'Geo package',
-                'file_extension': 'gpkg'
-            },
-            'shp': {
-                'name': 'Shape file',
-                'file_extension': 'shp'
-            }
-        }
 
-    @staticmethod
-    def export_options():
-        return {
-            # has to be implemented next:
-            # 'coordinate_reference_system': {
-            #     'label': 'Coordinate reference system',
-            #     'type': 'choice',
-            #     'default': 'pseudomerkator',
-            #     'groups': [
-            #         {
-            #             'name': 'Global coordinate reference systems',
-            #             'values': [
-            #                 {'name': 'pseudomerkator', 'label': 'Pseudo merkator'},
-            #                 {'name': 'wgs72', 'label': 'WGS 72'},
-            #                 {'name': 'wgs84', 'label': 'WGS 84'}
-            #             ]
-            #         },
-            #         {
-            #             'name': 'UTM zones for your export',
-            #             'values': [
-            #                 {'name': 'utm32', 'label': 'UTM zone 32'},
-            #                 {'name': 'utm33', 'label': 'UTM zone 33'}
-            #             ]
-            #         }
-            #     ]
-            # },
-            # 'detail_level': {
-            #     'label': 'Detail level',
-            #     'type': 'choice',
-            #     'default': 'verbatim',
-            #     'values': [
-            #         {'name': 'verbatim', 'label': 'Verbatim'},
-            #         {'name': 'simplified', 'label': 'Simplified'}
-            #     ]
-            # }
-        }
+def converter_configuration():
```

Comment:
Another function to wrap another function without doing really anything, seems a bit repetitious to me - is it correct, that this doesn't get used at all (except in the tests)? Then I'd argue this'd be moved there.